src/chainer.test.js

```diff
@@ -1 +1,50 @@
-const { chainer } = require('./chainer');
+'use strict';
+
+describe('chainer', () => {
+  const { chainer } = require('./chainer');
+
+  it(`should correctly chain functions and return the expected result`, () => {
+    const addOne = (x) => x + 1;
+    const multiplyByTwo = (x) => x * 2;
+    const subtractThree = (x) => x - 3;
+
+    const chainedFunctions = chainer([addOne, multiplyByTwo, subtractThree]);
+
+    expect(chainedFunctions(5)).toEqual(9);
+  });
+
+  it(`should correctly chain functions and return the expected'
```

Comment:
It's better to keep the test case descriptions on one line for readability. Avoid splitting it into two lines.